### ISSUE TITLE
Video caption and language buttons do not have standard focus highlight

### ISSUE THREAD

**Issue by bjester** [learningequality/kolibri#6035]
### Observed behavior

Video.js defines styling that highlights an enabled menu/button in the control bar when it's focused. The new caption and caption language buttons accompanying the transcript feature are missing this highlight.

| *Control* | Volume control | Caption control | Caption language control |
| --- | --- | --- | --- |
| *Status* | Present | Missing | Missing |
| *Screenshot* | ![screenshot-35 197 126 33-8080-2019 11 07-13-44-19](https://user-images.githubusercontent.com/819838/68430566-23abae80-0165-11ea-87f5-bfa5728a665e.png) | ![screenshot-35 197 126 33-8080-2019 11 07-13-44-56](https://user-images.githubusercontent.com/819838/68430590-2e664380-0165-11ea-9b6b-78b4f8b6fdbd.png) | ![screenshot-35 197 126 33-8080-2019 11 07-13-45-10](https://user-images.githubusercontent.com/819838/68430602-39b96f00-0165-11ea-9c42-1529e00e4f84.png) |

### Expected behavior

Controls should have some visual indication of which is active, which control the menu relates to.

### User-facing consequences

There may be confusion as to which menu is enabled.

### Steps to reproduce

1) Load a video in Kolibri
2) Click on CC or Languages control button in video control bar
3) Observe absence of highlight behind control button icon

### Context

Kolibri 0.13 https://github.com/learningequality/kolibri/pull/5631
Occurs in any browser

**Comment by indirectlylit** [comment]
I don't know about the technical complexity, but when tackling this, we should override the default video.js focus behaviors and instead using our standard default outline for all controls.

The current video.js behavior is fairly subtle and low-contrast:

![image](https://user-images.githubusercontent.com/2367265/68438964-eca0c180-0193-11ea-861f-79979022e9d0.png)

**Comment by radinamatic** [comment]
Completely in agreement with @indirectlylit here. 
(_I remember adding more prominent focus ring styles to video.js we had in KA Lite 4 years ago when I started._)

Let's stick with our standard outline, which is already in your PR :slightly_smiling_face:

**Comment by pcenov** [comment]
@radinamatic this is still extant in 0.15.1

**Comment by marcellamaki** [comment]
Closed with https://github.com/learningequality/kolibri-design-system/pull/185
